android/app/build.gradle

```diff
@@ -127,9 +127,9 @@ android {
     }
 }
 
-// Excludes duplicate inclusions of OkHttp - from SalesforceSDK and React.
-configurations {
-    all*.exclude group: 'com.squareup.okhttp3'
+// Excludes duplicate inclusions of OkHttp - from SalesforceSDK and React. Forces OkHttp 3.4.1 to be used.
+configurations.all {
+    resolutionStrategy.force 'com.squareup.okhttp3:okhttp:3.4.1'
```

Comment:
@ivanbogdanov My previous 'fix' was excluding both libs :-(
This one forces it to use `3.4.1` as the resolution strategy. The app runs now without crashing.